### ISSUE TITLE
The exception wrapper in solvcon.cmdutil.go is not useful

### ISSUE THREAD

**Issue by yungyuc** [solvcon/pastsolvcon#100]
Originally reported by: **Yung-Yu Chen (BitBucket: [yungyuc](http://bitbucket.org/yungyuc), GitHub: [yungyuc](http://github.com/yungyuc))**

---

The exception wrapper doesn't provide useful error messages.  It should be either:
1. removed, or
2. more developed to give useful information.

The example error message:

```
#!python

*** *** Traceback (most recent call last):
  File "./go", line 338, in <module>
    solvcon.go()
  File "/path/solvcon/solvcon/cmdutil.py", line 188, in go
    cmd()
  File "/path/solvcon/solvcon/command.py", line 1091, in __call__
    func(submit=False, **funckw)
  File "/path/solvcon/solvcon/case.py", line 907, in simu
    cse.run(level=runlevel)
  File "/path/solvcon/solvcon/case.py", line 663, in run
    self._run_preloop()
  File "/path/solvcon/solvcon/case.py", line 697, in _run_preloop
    self.solver.solverobj.preloop()
  File "/path/solvcon/solvcon/solver.py", line 305, in preloop
    self.runanchors('preloop')
  File "/path/solvcon/solvcon/anchor_legacy.py", line 131, in __call__
    func()
  File "/path/solvcon/solvcon/kerpak/cese.py", line 612, in preloop
    for point in self.points: point.locate_cell(self.svr)
  File "/path/solvcon/solvcon/kerpak/cese.py", line 580, in locate_cell
    idx = svr.locate_point(*self.crd)
AttributeError: ('run.__call__()', "'GasSolver' object has no attribute 'locate_point'")
```

---
- Bitbucket: https://bitbucket.org/solvcon/solvcon/issue/100

**Comment by yungyuc** [comment]
_Original comment by_ **Yung-Yu Chen (BitBucket: [yungyuc](http://bitbucket.org/yungyuc), GitHub: [yungyuc](http://github.com/yungyuc))**:

---

Ref issue #100 remove the useless exception handler.

**Comment by yungyuc** [comment]
_Original comment by_ **Yung-Yu Chen (BitBucket: [yungyuc](http://bitbucket.org/yungyuc), GitHub: [yungyuc](http://github.com/yungyuc))**:

---

Resolved by pull request #48.

I can't think of better error messages now.  Just don't process the exception.
